lib/plug/builder.ex

```diff
@@ -212,11 +212,21 @@ defmodule Plug.Builder do
       :function -> "expected #{inspect plug}/2 to return a Plug.Conn"
     end
 
+    halt_message = case plug_type do
+      :module   -> "plug pipeline halted in #{inspect plug}.call/2"
```

Comment:
What if we replace "plug pipeline" by `#{inspect module}`? Something like:

    #{inspect env.module} halted on plug #{inspect plug}.call/2